arviz/plots/ppcplot.py

```diff
@@ -127,7 +127,9 @@ def plot_ppc(
 
         >>> import arviz as az
         >>> data = az.load_arviz_data('radon')
-        >>> az.plot_ppc(data)
+        >>> az.plot_ppc(data,data_pairs={"obs":"obs"})
+        >>> #az.plot_ppc(data,data_pairs={"obs":"obs_hat"})
```

Comment:
This commented out line should be deleted as well

Reply:
That was my suggestion, do you have other way to include that example?

Reply:
I would leave the comment here as a remainder of including PyStan examples